pulp_ansible/app/models.py

```diff
@@ -417,3 +436,8 @@ class AnsibleDistribution(Distribution):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+
+        permissions = [
+            ("manage_roles_ansibledistribution", "Can manage roles on ansible distributions"),
+            ("modify_ansible_repo_content", "Can modify ansible repository content"),
```

Comment:
I would just defer to the repository permissions for managing the content of the repository.

Reply:
I agree, let's try to reduce the duplicate permissions.